lib/errors.js

```diff
@@ -44,11 +45,17 @@ module.exports = (options) => {
     const translate = opts.translate || req.translate;
     const content = getContent(err, translate);
 
+    let template = 'error';
+
+    if (res.locals.partials && res.locals.partials[err.status]) {
+      template = err.status.toString();
+    }
```

Comment:
This isn't a reliable way of testing if a template is renderable. Not all of the directories in `app.get('views')` will necessarily be mounted as partials.

The better way to try to load the template as per https://github.com/UKHomeOfficeForms/hof-middleware-markdown/blob/master/lib/markdown.js#L30-L38, which is taken from what express itself does - https://github.com/expressjs/express/blob/master/lib/application.js#L568-L583

Given that, the best way to do this might just be to try to render the template, and if that fails, render a default.